### PR TITLE
Implemented: picker editing on in-progress orders page (#237)

### LINKED ISSUE #237: Enable Picker editing on the In Progress Orders Page
## What is the motivation for adding/enhancing this feature?
Users should be able to edit the pickers on the In Progress Orders page. They should have the flexibility to reassign pickers or update picker information.

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

## Additional information
Please find the [UI](https://www.figma.com/file/wK5FMs4z2zXQR9VfHV9ycE/Dhiraj's-Ionic-6-Material-UI-Kit-(Community)?type=design&node-id=27543%3A76557&mode=design&t=7CPWHNaQZoMIFvQI-1)

### PR DESCRIPTION
**Pull request by k2maan**
### Related Issues

Closes #237 

### Screenshots of Visual Changes before/after (If There Are Any)

[Screencast from 21-08-23 12:43:57 PM IST.webm](https://github.com/hotwax/fulfillment-pwa/assets/59442907/94888133-408b-45ae-9846-61782557d5c1)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/ionic-bopis#contribution-guideline)

### CODE REVIEW

File: src/components/EditPickersModal.vue [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    isPickerSelected(id: string) {
+      return this.selectedPickers.some((picker: any) => picker.id == id)
+    },
+    selectPicker(id: string) {
+      const picker = this.selectedPickers.some((picker: any) => picker.id == id)
+      if (picker) {
+        // if picker is already selected then removing that picker from the list on click
+        this.selectedPickers = this.selectedPickers.filter((picker: any) => picker.id != id)
+      } else {
+        this.selectedPickers.push(this.pickers.find((picker: any) => picker.id == id))
```
Comment (ymaheshwari1) [inline src/components/EditPickersModal.vue:118]: We need to discuss how the system generated picklist will be handled.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/EditPickersModal.vue (hunk 1 above)
Comment (k2maan) [reply src/components/EditPickersModal.vue:118]: Implemented changes as discussed

File: src/views/InProgress.vue [hunk 2]
```diff
@@ -164,6 +170,22 @@
         <p v-html="getErrorMessage()"></p>
       </div>
     </ion-content>
+    <!-- only show footer buttons if 'All orders' is not selected -->
+    <ion-footer v-if="selectedPicklistId.length">
```
Comment (ymaheshwari1) [inline src/views/InProgress.vue:174]:
```suggestion
    <ion-footer v-if="selectedPicklistId">
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/InProgress.vue (hunk 2 above)
Comment (k2maan) [reply src/views/InProgress.vue]: Updated sir

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**